src/lib/components/FornaContainer.react.js

```diff
@@ -0,0 +1,239 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {FornaContainer as PreFornaContainer} from 'fornac';
+import * as R from 'ramda';
+
+/**
+ * This is a FornaContainer component.
+ */
+export default class FornaContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.renderNewSequences = this.renderNewSequences.bind(this);
+    }
+
+    componentDidMount() {
+        const {height, width} = this.props;
+
+        this._fornaContainer = new PreFornaContainer(this.ref, {
+            initialSize: [width, height],
+        });
+        this.renderNewSequences();
+    }
+
+    componentDidUpdate() {
+        this.renderNewSequences();
+    }
+
+    renderNewSequences() {
+        const {sequences} = this.props;
+
+        if (this._fornaContainer) {
+            this._fornaContainer.clearNodes();
+
+            sequences.forEach(seq => {
+                const unpackedOptions = Object.assign({}, seq.options, {
+                    sequence: seq.sequence,
+                    structure: seq.structure,
+                });
+                this._fornaContainer.addRNA(seq.structure, unpackedOptions);
+            });
+        }
+    }
+
+    shouldComponentUpdate(nextProps) {
+        const {sequences, colorScheme, nodeFillColor} = this.props;
+
+        if (!R.equals(sequences, nextProps.sequences)) {
+            return true;
+        }
+        this._fornaContainer.addCustomColors(nextProps.customColors);
+
+        if (nodeFillColor !== nextProps.nodeFillColor) {
+            this._fornaContainer.setOutlineColor(nextProps.nodeFillColor);
+        }
+
+        this._fornaContainer.changeColorScheme(colorScheme);
+
+        return false;
+    }
+
+    render() {
+        return (
+            <div
+                id={this.props.id}
+                ref={ref => {
+                    this.ref = ref;
+                }}
```

Comment:
FYI this works - but there's a newer way to do this with `react.createRef` https://reactjs.org/docs/refs-and-the-dom.html#creating-refs